src/Components/BureauPage/ProjectedVacancy/projectedVacancy.jsx

```diff
@@ -204,6 +204,8 @@ const ProjectedVacancy = () => {
     includeSelectAll: true,
   };
 
+  const dummyid = get(dummyPositionDetails, 'id', '');
```

Comment:
i know this will be removed anyways, but just to get us into the habit of not using Lodash, here is the vanilla js alternative: 
```suggestion
  const dummyid = dummyPositionDetails?.id;
```